input.yml

```diff
@@ -99,10 +99,10 @@
       - name: DNS adblocking prompt
         pause:
           prompt: |
-            Do you want to install an ad blocking DNS resolver on this VPN server?
+            Do you want to configure the DNS ad blocking on this VPN server?
```

Comment:
I would suggest "do you want to enable DNS ad blocking" instead. "Configure" can be ambiguous.